cmd/miniooni/main.go

```diff
@@ -273,20 +274,16 @@ func main() {
 		defer experiment.CloseReport(ctx)
 	}
 
-	for _, input := range inputs {
+	inputCount := len(globalOptions.inputs)
+	inputCounter := 0
+	for _, input := range globalOptions.inputs {
+		inputCounter++
+		log.Infof("[%d/%d] running with input: %s",inputCounter,inputCount,input)
 		measurement, err := experiment.Measure(ctx, input)
 		if err != nil {
 			log.WithError(err).Warn("measurement failed")
 			continue
 		}
-		// Remember to omit the user IP.
-		//
-		// This is not ooniprobe and there's no need here to increase the
-		// complexity to make this option configurable. Still, I don't want
-		// this tool to be sharing the IP, because I want to provide the
-		// same default level of sharing of ooniprobe to random people that
-		// may run this tool for development purposes or exploration.
-		measurement.ProbeIP = model.DefaultProbeIP
```

Comment:
I do not understand where this part of the diff comes from. It seems to me it should not be here?

Reply:
I ruined everything by one mistake. -_-